buildenv/jenkins/common/pipeline-functions.groovy

```diff
@@ -606,6 +606,18 @@ def generate_test_jobs(TARGET_NAMES, SPEC, ARTIFACTORY_SERVER, ARTIFACTORY_REPO)
     }
 }
 
+def retry_and_delay(command, numRetries = 3, waitTime = 60, units = 'SECONDS') {
+    def ret = false
+    retry(numRetries) {
+    if (ret) {
+        sleep time: waitTime,unit: units
```

Comment:
Whitepsace after comma.

Reply:
Thanks for find those mistakes! I have fixed them. Thanks for checking it again~